docs/docsource/Operation-Architecture/Services/lifecycle_management/lifecycle_expiration_policies.rst

```diff
@@ -1,40 +1,42 @@
 Lifecycle Expiration Policies
 =============================
 
-Cloud users can apply lifecycle expiration rules (specified in Amazon’s
-`AWS S3 API <https://docs.aws.amazon.com/AmazonS3/latest/API/Welcome.html>`__)
+Cloud users can apply lifecycle expiration or transition rules (specified in 
+Amazon’s `AWS S3 API <https://docs.aws.amazon.com/AmazonS3/latest/API/Welcome.html>`__)
 to buckets managed through Zenko. These rules are triggered after a defined
-time has passed since the object’s creation.
+time has passed since the object’s creation. For the purposes of lifecycle 
+management, an object is "created" when it is touched.
```

Comment:
What does this last sentence mean? x)

Reply:
I was using "touched" in what I understand to be the UNIX sense; i.e., the object undergoes any change that the OS would pick up on (this is normally associated with a date change on the object). Am I misusing this term?

The big idea is that if the object is changed in any way, Zenko considers it a "new" object and resets the clock to zero.

Reply:
Per Bennet: "These rules are triggered after a defined time has passed since the object’s last modification." 

Full stop.